core/workspace_svg.js

```diff
@@ -382,9 +401,22 @@ Blockly.WorkspaceSvg.prototype.resize = function() {
   if (this.scrollbar) {
     this.scrollbar.resize();
   }
+  this.updateScreenCalculations_();  
+};
 
-  this.updateInverseScreenCTM();
-  this.recordDeleteAreas();
+/**
+ * Resizes and repositions workspace chrome if the page has a new
+ * scroll position.
+ * @package
+ */
+Blockly.WorkspaceSvg.prototype.updateScreenCalculationsIfScrolled
+  = function () {
```

Comment:
Done! Had to use the disable indent trick.  Considered renaming the function instead... but couldn't come up with anything better :)